src/index.js

```diff
@@ -135,7 +135,7 @@ class Pathfinding {
 		const nodes = this.zones[zoneID].groups[groupID];
 		const vertices = this.zones[zoneID].vertices;
 
-		const closestNode = this.getClosestNode(startPosition, zoneID, groupID);
+		const closestNode = this.getClosestNode(startPosition, zoneID, groupID, true);
```

Comment:
Not sure why this was false in the first place. Spent some time pathing around in the demo scene and it seems setting it to true did not break anything.